### PR TITLE
something terrible, just to test

### PR DESCRIPTION
**Pull request by aliliolek**
[DEMO LINK](https://aliliolek.github.io/react_autocomplete/)

### CODE REVIEW

File: src/App.tsx
Comment (maxim2310) [inline src/App.tsx:16]: you must not select person by default 
image

File: src/App.tsx
Comment (maxim2310) [inline src/App.tsx:28]: make this delay is smaller, about 0.8s or 0.5s

File: src/App.tsx
Comment (maxim2310) [inline src/App.tsx:74]: to much actions, move to separate func

File: src/App.tsx
Comment (maxim2310) [inline src/App.tsx:82]: why do you need classnames here?

File: src/App.tsx
Comment (maxim2310) [inline src/App.tsx:154]:
```suggestion
          className="dropdown-menu dropdown-menu--height"
```

**Review by maxim2310 — CHANGES_REQUESTED** [review]
Great job overall! Just couple tiny things needs to be fixed before approval)

File: src/App.tsx
Comment (mykhalenych) [inline src/App.tsx:2]: Screenshot 2023-09-05 at 17 31 28
you are using lodash but didn't install module in package.json
also try to write your own debounce function

File: src/App.tsx
Comment (mykhalenych) [inline src/App.tsx:24]: why do you need set addition state after toggle?
`const isActive = clickCount % 2 === 0`

**Review by mykhalenych — CHANGES_REQUESTED** [review]
Nice work, but fix small suggestions

**Pull request closed without merging**
